lucene/suggest/src/java/org/apache/lucene/search/suggest/document/TopSuggestDocs.java

```diff
@@ -105,6 +105,10 @@ public TopSuggestDocs(TotalHits totalHits, SuggestScoreDoc[] scoreDocs, boolean
     this.isComplete = isComplete;
   }
 
+  public TopSuggestDocs(TotalHits totalHits, SuggestScoreDoc[] scoreDocs) {
```

Comment:
I'm not sure about how bwc is handled in Lucene, especially if we intend to backport this to maybe 8.x, so I don't know if we need a ctor without the isComplete flag?

Reply:
Since the javadocs for this class are marked with the `@lucene.experimental` tag, we are allowed to change the API even in the next dot release (8.x), so it is not necessary to add telescoping constructors here, so we can keep only the constructor above that takes the new `boolean isComplete`.